libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ChartConfigurationFlyoutUtils.tsx

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { IDropdownOption } from "@fluentui/react";
+import { ErrorCohort } from "@responsible-ai/core-ui";
+
+import { IChartConfigurationFlyoutProps } from "./ChartConfigurationFlyoutProps";
+
+export const selectAllOptionKey = "selectAll";
+
+export function getInitialNewlySelectedDatasetCohorts(
+  props: IChartConfigurationFlyoutProps
```

Comment:
can we just pass the props that are needed to this function and the following function? this avoids the overhead of passing all unwanted props.

Reply:
good point, done!